scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -1404,3 +1404,30 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-warning">Part
   </td>
 </tr>
 {% endmacro %}
+
+{% block scripts %}
+  {{ super() }}
+  <script>
+    function printPDF(){
+      {% if config.SQLALCHEMY_DATABASE_URI %} // include coverage report
+        fetch('{{ url_for('cases.case_coverage_data', institute_id=institute._id, case_name=case.display_name) }}', {
+          method: 'GET',
+          headers: {'Content-Type': 'application/json'}
+          })
+          .then(resp => resp.text())
+          .then(html => {
+            document.body.innerHTML = document.body.innerHTML + "<hr>" + html;
```

Comment:
Added a js function that adds the coverage data to the report when you print the PDF, I don't know if it works tho, I'll test it!